lms/djangoapps/discussion/rest_api/api.py

```diff
@@ -333,7 +363,9 @@ def get_non_courseware_topics(request, course_key, course, topic_ids):
     for name, entry in sorted_topics:
         if not topic_ids or entry['id'] in topic_ids:
             discussion_topic = DiscussionTopic(
-                entry["id"], name, get_thread_list_url(request, course_key, [entry["id"]])
+                entry["id"], name, get_thread_list_url(request, course_key, [entry["id"]]),
+                None,
+                thread_counts.get(entry["id"], {"discussion": 0, "question": 0})
```

Comment:
As I commented above, it would be easier if the default value is set in the `DiscussionTopic` class.